pulsar-broker/src/test/java/org/apache/pulsar/client/api/ClientDeduplicationFailureTest.java

```diff
@@ -60,7 +59,7 @@
 import org.testng.annotations.Test;
 
 @Slf4j
-@Test(groups = "quarantine")
+@Test(groups = "broker")
```

Comment:
Was this added in #16605? before taking tests out of quarantine, it should be ensured that the test is not flaky any more. The problem fixed in this PR isn't about flakiness. Have you verified that ClientDeduplicationFailureTest is no more flaky?

Reply:
How to verify? Do we have any rules for leaving the flaky test?

Reply:
Maybe we can change the test group in another PR.

@lhotari I think it's hard to judge whether the test is flaky or not, maybe the flaky problem was fixed in some PRs. Maybe we need to find a way to observe the failed frequency of the flaky tests.

Reply:
We can discuss it in another thread. I will roll back the test group.

Reply:
Thanks, @mattisonchao, in fact, if we can't reproduce the problem, it seems it's hard to judge whether the test is flaky or not, maybe run hundreds of times, and the problem presents a few times.